README.md

```diff
@@ -1,7 +1,6 @@
 Tweepy: Twitter for Python!
 ======
 
-[![Build Status](http://img.shields.io/travis/tweepy/tweepy/master.svg?style=flat)](https://travis-ci.org/tweepy/tweepy)
```

Comment:
This could be replaced with the status badge for the Test workflow:
```suggestion
![Test Status](https://github.com/tweepy/tweepy/workflows/Test/badge.svg)
```